src/main/java/racingcar/model/RaceGame.java

```diff
@@ -0,0 +1,47 @@
+package racingcar.model;
+
+import java.util.List;
+import camp.nextstep.edu.missionutils.Randoms;
+import racingcar.view.OutputView;
+
+public class RaceGame {
+    private final Cars cars;
+    private final int raceCount;
+    private static final int MINIMUM_MOVE_THRESHOLD = 4;
+
+    public RaceGame(Cars cars, int raceCount) {
+        this.cars = cars;
+        this.raceCount = raceCount;
+    }
+    public void moveCars() {
+        for (Car car : cars.getCars()) {
+            if (canMove()) cars.incrementCarPosition(car);
+        }
```

Comment:
```suggestion
        // RaceGame에서 Cars를 거치지 않고 직접 Car를 조종하거나
        cars.getCars().stream()
            .filter(car -> canMove())
            .forEach(Car::incrementPosition);

        // Cars의 incrementPosition(car)를 통해 간접적으로 조종
        cars.getCars().stream()
            .filter(car -> canMove())
            .forEach(cars::incrementCarPosition);
```

이렇게 하면 가독성을 조금이나마 높일 수 있을 것 같아요!!

Reply:
제 이전 코드는 직접 조종하다 보니 코드는 짧되 가독성이 떨어지는데, 가독성과 함께 스트림을 사용할 수 있어서 제시해 주신 코드가 더욱 나은 코드라고 생각됩니다...... 피드백 정말 감사드려요 indept 2를 신경 쓰다 보니 메서드를 만들고, 기존 제 방식보다 코드 길이를 줄이느라 가독성을 미처 신경 쓰지 못한 것 같아요